spiped/dispatch.c

```diff
@@ -23,6 +23,8 @@ struct accept_state {
 	int requirefps;
 	int nokeepalive;
 	int * conndone;
+	int only_one;
+	int shutdown;
```

Comment:
I'd call this `shutdown_requested`; the name `shutdown` is ambiguous (is it an adjective or a verb?).